kafka-ui-react-app/src/components/App.styled.ts

```diff
@@ -42,6 +43,28 @@ export const Sidebar = styled.div<{ $visible: boolean }>(
       left: -${theme.layout.navBarWidth};
       z-index: 100;
     }
+
+    &::-webkit-scrollbar {
+      width: 8px;
+    }
+
+    &::-webkit-scrollbar-track {
+      background-color: ${Colors.neutral[0]};
```

Comment:
We should not use the Colors map directly. All colors should be defined in the theme. Colors is just a palette. 
```suggestion
      background-color: ${theme.layout.scrollbar.background}; 
```